staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/time.go

```diff
@@ -130,7 +130,7 @@ func (t *Time) UnmarshalQueryParameter(str string) error {
 		return nil
 	}
 
-	pt, err := time.Parse(time.RFC3339, str)
```

Comment:
Old servers will not have this change. Changing clients to serialize in a way old servers cannot deserialize breaks compatibility.

Reply:
see https://github.com/kubernetes/kubernetes/pull/92595#issuecomment-652160810